### ISSUE TITLE
Supporting Plots.jl

### ISSUE THREAD

**Issue by fbruetting** [JuliaGraphics/Immerse.jl#75]
Would you like this package to support Plots.jl with all (or at least some) of it’s back-ends in the future, too?

**Comment by timholy** [comment]
Yes, broader plotting support would be great, and interactivity at the level of this package seems largely missing from other options. I started this package at a time when Gadfly seemed likely to become the standard, and Plots didn't even exist (IIRC).
